overlord/managers_test.go

```diff
@@ -3498,6 +3523,27 @@ version: 2.0`
 	c.Assert(tasks, HasLen, i+1)
 }
 
+func (ms *mgrsSuite) mockSuccessfulSystemRestart(c *C, bloader *bootloadertest.MockBootloader) {
```

Comment:
should we use /SystemRestart/Reboot/ ?

Reply:
they seem important enough helper to go before the first tests using them, not after, but just my 2c

Reply:
we can also use st.Restarting() to check whether there is a restart of the right kind going on